benchmarks/array/mapify.bench.ts

```diff
@@ -0,0 +1,27 @@
+import * as _ from 'radashi'
+import { bench } from 'vitest'
+
+describe('mapify', () => {
+  bench('with full list and identity value function', () => {
+    const list = [
+      { id: 'a', word: 'hello' },
+      { id: 'b', word: 'bye' },
+      { id: 'c', word: 'oh' },
+      { id: 'd', word: 'hey' },
+      { id: 'e', word: 'ok' },
+    ]
+    _.objectify(
```

Comment:
Still using `objectify` in the benchmarks

Reply:
fixed